### PR TITLE
Minor Qt GUI update

### PR DESCRIPTION
**Pull request by Xphalnos**
I did a little code cleanup and updated the dialogs.
The build size has been reduced by 0.2MB.
I reversed the dark theme and the light theme because the majority of users use dark mode.

### CODE REVIEW

**Comment by raphaelthegreat** [comment]
Can you show some screenshot of changes?

**Comment by Xphalnos** [comment]
As I said, these are minor changes!

OLD :
Capture d'écran 2024-08-04 135901

NEW :
Capture d'écran 2024-08-04 135716


OLD :
Capture d'écran 2024-08-04 135952

NEW :
Capture d'écran 2024-08-04 140045

File: src/qt_gui/gui_context_menus.h [hunk 1]
```diff
@@ -165,12 +165,12 @@ class GuiContextMenus : public QObject {
                     if (createShortcutLinux(linkPath, ebootPath, iconPath)) {
 #endif
                         QMessageBox::information(
-                            nullptr, "Shortcut Creation",
-                            QString("Shortcut created successfully:\n %1").arg(linkPath));
+                            nullptr, "Shortcut creation...",
+                            QString("Shortcut created successfully !\n %1").arg(linkPath));
```
Comment (polybiusproxy) [inline src/qt_gui/gui_context_menus.h]:
```suggestion
                            QString("Shortcut created successfully!\n %1").arg(linkPath));
```
Having a space before the exclamation mark is a grammatical error, same thing on the rest of strings

File: src/qt_gui/game_info.cpp [hunk 2]
```diff
@@ -25,7 +26,7 @@ void GameInfoClass::GetGameInfo(QWidget* parent) {
               }).results();
 
     // Progress bar, please be patient :)
-    QProgressDialog dialog("Loading game list, please wait :3", "Cancel", 0, 0, parent);
+    QProgressDialog dialog("Loading game list...", "Cancel", 0, 0, parent);
```
Comment (raphaelthegreat) [inline src/qt_gui/game_info.cpp]: The old text had more personality I think xD

File: src/qt_gui/gui_context_menus.h [hunk 3]
```diff
@@ -165,12 +165,12 @@ class GuiContextMenus : public QObject {
                     if (createShortcutLinux(linkPath, ebootPath, iconPath)) {
 #endif
                         QMessageBox::information(
-                            nullptr, "Shortcut Creation",
-                            QString("Shortcut created successfully:\n %1").arg(linkPath));
+                            nullptr, "Shortcut creation...",
```
Comment (raphaelthegreat) [inline src/qt_gui/gui_context_menus.h]: 3 dots imply some form of progress so not super fitting for window title

File: src/qt_gui/game_info.cpp (hunk 2 above)
Comment (Xphalnos) [reply src/qt_gui/game_info.cpp]: Do you want me to keep it?

Comment (raphaelthegreat) [reply src/qt_gui/game_info.cpp]: Yes

**Comment by Xphalnos** [comment]
Done!

**Review by polybiusproxy — APPROVED** [review]

**Pull request merged**